corehq/ex-submodules/casexml/apps/case/tests/test_rebuild.py

```diff
@@ -509,6 +509,30 @@ def _test(self, action_tuples):
         for index, action in action_tuples:
             self.assertEqual(action, sorted_actions[index])
 
+
+class CaseRebuildSimpleTest(SimpleTestCase):
+
+    def test_rebuild_with_opened_on_in_update(self):
+        case = CommCareCase(
+            actions=[
+                CommCareCaseAction.wrap({
+                    "xform_id": "8166142bceefae35375abfdf285f9dfc",
+                    "user_id": "66925160e15a21bc6f332e967dc05647",
+                    "server_date": "2013-02-07T04:40:40Z",
+                    "action_type": "update",
+                    "updated_known_properties": {
+                        "opened_on": "2012-10-18T12:16:22Z",
```

Comment:
@dannyroberts is the issue here the same as https://github.com/dimagi/dimagi-utils/pull/321? and if so, is it just a total coincidence that it happened to pop up at right around the same time?